src/ui.js

```diff
@@ -0,0 +1,72 @@
+import Storage from './storage.js';
+import TaskList from './tasklist.js';
+
+const todoList = document.querySelector('.todo-list');
+
+export default class UI {
+  static addToUI(task) {
+    const todoDiv = document.createElement('div');
+    todoDiv.classList.add('todo');
+    const todoCheck = document.createElement('input');
+    todoCheck.type = 'checkbox';
+    todoCheck.classList.add('todo-check');
+    if (task.checked === true) {
+      todoCheck.checked = true;
+      todoDiv.classList.add('checked');
+    }
+    todoDiv.appendChild(todoCheck);
+
+    const todoLi = document.createElement('li');
+    todoLi.classList.add('todo-item');
+    todoLi.innerText = task.description;
+    todoDiv.appendChild(todoLi);
+
+    const todoDots = document.createElement('button');
+    todoDots.classList.add('todo-btn');
+    todoDots.innerHTML = '<i class="fas fa-ellipsis-v"></i>';
+    todoDiv.appendChild(todoDots);
+    todoList.appendChild(todoDiv);
+    UI.checkTask();
+  }
+
+  static displayList = (arr) => {
+    arr.forEach((task) => {
+      UI.addToUI(task);
+      UI.checkTask();
+    });
+  };
+
+  static checkTask() {
+    document.querySelectorAll('.todo-check').forEach((checkbox) => {
+      checkbox.addEventListener('change', (e) => {
+        if (e.target.checked) {
+          const newList = new TaskList();
+          e.target.parentElement.classList.add('checked');
+          Storage.getList().list.forEach((task) => {
```

Comment:
- [ ] When I tried to add multiple similar tasks in the TODO list and marking a single todo out of the similar todo task it works. But after that when i try to refresh the page, all other tasks with similar description also gets marked. Kindly try to fix this issue.

- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

Todo marked
![image](https://user-images.githubusercontent.com/53019916/130235620-9ab37d2f-ecf3-43cc-aaad-03f5be13c493.png)

On Local storage
![image](https://user-images.githubusercontent.com/53019916/130235684-2e927165-730f-4e54-a31d-c5bd9323fea6.png)